account_invoice_pricelist/README.rst

```diff
@@ -0,0 +1,90 @@
+===============================
```

Comment:
You can remove this file. ocabot will auto create it for v13 when merged.

Reply:
Done